### PR TITLE
[WIP]  changes/fixes for azure release

### PR DESCRIPTION
**Pull request by SauarvD**
### Requirements

* Changes/Fixes for azure release OSS
### Description of the Change

* Changes in build pack website for static/angular/react
* Changes in azure function groovy module to handle responses and errors
* Changes in index.js of azure create service.

### Benefits

Users can now deploy their services on azure platform as well

### CODE REVIEW

File: builds/jazz_azure-create-service/index.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
```
Comment (devsatishm) [inline builds/jazz_azure-create-service/index.js]: Looks like this module uses deprecated azure modules. Let's check if we can use the new azure node sdk - https://github.com/azure/azure-sdk-for-js
```
############################################################################
# Active development of this SDK has been moved to [31m@azure/arm-cdn[0m[1m package. #
# This package is in maintenance mode and will be deprecated in July 2019. #
#  Visit https://aka.ms/azure-sdk-for-js-migration for more information.   #
############################################################################

[0m
[1m

#################################################################################
# Active development of this SDK has been moved to [31m@azure/arm-eventhub[0m[1m package. #
#   This package is in maintenance mode and will be deprecated in July 2019.    #
#     Visit https://aka.ms/azure-sdk-for-js-migration for more information.     #
#################################################################################

[0m
[1m

###################################################################################
# Active development of this SDK has been moved to [31m@azure/arm-servicebus[0m[1m package. #
#    This package is in maintenance mode and will be deprecated in July 2019.     #
#      Visit https://aka.ms/azure-sdk-for-js-migration for more information.      #
###################################################################################```

Comment (devsatishm) [reply builds/jazz_azure-create-service/index.js]: Following are the modules that might need to be migrated - 

```
    "azure-arm-apimanagement": "^4.4.0-preview",
    "azure-arm-cdn": "^4.1.0",
    "azure-arm-cosmosdb": "^2.3.0",
    "azure-arm-eventhub": "^3.1.0",
    "azure-arm-resource": "^7.3.0",
    "azure-arm-sb": "^3.1.0",
    "azure-arm-storage": "^7.0.0",
    "azure-arm-website": "^5.7.0",
    "azure-storage": "^2.10.2"
```

File: builds/jazz-build-module/azure-function-deploy-module.groovy [hunk 2]
```diff
@@ -49,7 +49,7 @@ def sendAssetCompletedEvent(serviceInfo, assetList) {
   data.tagValue = serviceInfo.stackName
   def output = azureUtil.invokeAzureService(data, "getResourcesByServiceName")
 
-  for (item in output.data.result) {
+  for (item in output.result) {
```
Comment (raghits) [inline builds/jazz-build-module/azure-function-deploy-module.groovy]: This change was breaking the build.

File: builds/jazz-build-module/azure-utility.groovy [hunk 3]
```diff
@@ -138,7 +138,7 @@ def invokeAzureService(data, command) {
   echo "azure service $command $output"
 
   def outputJson =  parseJson(output)
-  if (outputJson.data.error) {
+  if (outputJson && outputJson.data && outputJson.data.error) {
```
Comment (raghits) [inline builds/jazz-build-module/azure-utility.groovy]: Another change which was breaking the build.

File: builds/jenkins-build-pack-website/Jenkinsfile [hunk 4]
```diff
@@ -994,8 +1004,15 @@ def loadBuildModules(buildModuleUrl){
 
 
 
-def createWebsite(stackName, resourceGroup, storageName){
-  sh "cd ./app && zip -qr ../content.zip *  && cd .."
+def createWebsite(stackName, resourceGroup, storageName, framework){
```
Comment (raghits) [inline builds/jenkins-build-pack-website/Jenkinsfile]: This change was breaking Jenkins as it was zipping including node_modules etc. as it ran out of memory.

**Comment by SauarvD** [comment]
closed

**Pull request closed without merging**